documentation/environment-variables.md

```diff
@@ -22,5 +22,5 @@ For deployment in production there needs to be a way to change variables in the
 3) `docker-compose up --detach`
 
 To check if a particular environment variable is set in a container, a command like this could be used:
-`docker-compose exec php printenv FEATURE_DIRECTINTAKE`
```

Comment:
Oof, I need to update this entire file.

Reply:
This is why we need an evergreen docs site and add "update docs" as part of our definition of done :P